src/Microsoft.ML.TimeSeries/Deseasonality/StlDeseasonality.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace Microsoft.ML.TimeSeries
+{
+    internal sealed class StlDeseasonality : DeseasonalityBase
+    {
```

Comment:
All these `DeseasonalityBase` derived class implementations are really small. ML.NET does not have a restriction that there should be only one class per file. Can you please combine all these into a same file and remove the separate directory for Deseasonality?

Reply:
Updated.

---
In reply to: [442524532](https://github.com/dotnet/machinelearning/pull/5202#discussion_r442524532) [](ancestors = 442524532)